app/components/Dashboard/DashboardRecord/index.jsx

```diff
@@ -6,15 +6,22 @@ const SmartTimeDisplay = require('../../SmartTimeDisplay');
 require('./style.less');
 
 const UserStore = require('../../../stores/UserStore');
-export default React.createClass({
+const DashboardRecord = React.createClass({
```

Comment:
@RonaldoLee  @wizawu  @zycbobby 
Found, we'd better to give a name for each component, instead of anonymous.

Named components help React performance tool to divide the time cost to different components